### DISCUSSION TITLE
Не могу открыть образ

### DISCUSSION THREAD

**Discussion by egorf710** [extremecodetv/ExtremeCodeOS#178] [Q&A]
<html><body><p>Не удалось открыть образ виртуального диска <nobr><b>C:\ExtremeCodeOS-flat-1.0.0-Yobkiy.vmdk</b></nobr>.</p><qt><p>Could not get the storage format of the medium <b><nobr>'C:\ExtremeCodeOS-flat-1.0.0-Yobkiy.vmdk'</nobr></b> (VERR_NOT_SUPPORTED).</p>

Код ошибки: | VBOX_E_IPRT_ERROR (0x80BB0005)
-- | --
Компонент: | MediumWrap
Интерфейс: | IMedium {ad47ad09-787b-44ab-b343-a082a3f2dfb1}
Вызванный интерфейс: | IVirtualBox {d0a0163f-e254-4e5b-a1f2-011cf991c38d}
Код ошибки метода: | VBOX_E_OBJECT_NOT_FOUND (0x80BB0001)

<br></body></html>

**Comment by egorf710** [comment]
Попробовал открыть через VMware Player, но всё равно ошибка

**Comment by ghost** [comment]
У тебя проблемы с компом, у меня все работает без ошибок

**Reply by egorf710** [reply]
Можешь написать как ты устанавливал?

**Reply by ghost** [reply]
Тупо скачал образ и VMware. Открыл данный файл .vmdk в VMware и все. Попробуй переустановить виртуальные машины (VMware/VirtualBox), может поможет.

**Comment by Dasein-hub** [comment]
Мне бы тоже узнать, как эту божественную ОС ставить.

**Comment by hawr-desk** [comment]
Столкнулся с такой же проблемой. И как бы вопрос к людям, у которых всё получилось. У вас какая версия VirtualBox/VMware?

**Reply by ghost** [reply]
VirtualBox v6.1.28

**Reply by egorf710** [reply]
а какая ос?

**Reply by ghost** [reply]
Arch Linux 2021.11.01
